langlib/lang.array/src/main/ballerina/array.bal

```diff
@@ -130,6 +130,21 @@ public isolated function slice(Type[] arr, int startIndex, int endIndex = arr.le
     name: "slice"
 } external;
 
+# Tests whether a function returns true for every member of an array.
```

Comment:
Please update the docs similar to what's suggested in https://github.com/ballerina-platform/ballerina-lang/pull/35496.

Reply:
Changed